### ISSUE TITLE
Keybindings don't play nice on Win10 1809 + PSReadline 2.0

### ISSUE THREAD

**Issue by AndrewPla** [PowershellFrameworkCollective/PSUtil#42]
Looks like one of the keybindings scripts on 2.0.0 of PSUtil sets expand alias to spacebar. I can look into it more later, but figured I'd file the issue now. at first sight it appears like line 2 of expandAlias.ps1 at
`(Get-PSFConfigValue -FullName psutil.keybinding.expandalias -Fallback 'Shift+Spacebar') `

See below for an example of the behavior

```
PS C:\repos> 'look at spaces'
look at spaces
PS C:\repos> ipmo psutil
PS C:\repos> 'nospaces'
nospaces
PS C:\repos> Get-PSReadLineKeyHandler

#trimmed for brevity
User defined functions
======================

Key    Function      Description
---    --------      -----------
F1     CommandHelp   Open the help window for the current command
Ctrl+C CopyAllLines  Copies the all lines of the current command into the
                     clipboard
Space  ExpandAlias   Converts aliases into the resolved command / parameter
F7     History       Show command history
Alt+w  SaveInHistory Save current line in history but do not execute

```

**Comment by corbob** [comment]
Looks like this is related to this Issue in PSRL: https://github.com/lzybkr/PSReadLine/issues/719

To remove it without a reload: `Remove-PSReadLineKeyHandler` When prompted for chord to remove, hit the space bar. Hit `<enter>` twice and bam you can not hit space again 😃

**Comment by corbob** [comment]
FWIW: This is also an issue in PowerShell 6.1.1 on Linux (and probably on Windows :man_shrugging: )

**Comment by AndrewPla** [comment]
Related https://github.com/lzybkr/PSReadLine/issues/645

**Comment by FriedrichWeinmann** [comment]
Fixed in the `2.0.8` release
Will also apply the other related fix for PSReadline 2.0.0 default install with typing space while holding down shift
